setup.cfg

```diff
@@ -3,7 +3,9 @@ codestyle_max_line_length = 100
 codestyle_exclude = doc/conf.py
 addopts = --cov-report xml --cov=omc3
 testpaths = tests/unit tests/accuracy
-python_classes = BasicTests ExtendedTests
+markers=
+        basic: basic tests run for every commit
+        extended: test run on PRs
```

Comment:
just one of them should be enough

Reply:
should be enough but there is also no solid reason why to scrape it, is there? and anybody in the future might appreciate to find a reason why we distinguish